### ISSUE TITLE
want to convert subpages

### ISSUE THREAD

**Issue by bruceravel** [XraySpectroscopy/xafsorg#3]
Today I am again unable to get to millenia, so I cannot fetch things for myself from the moin site.

Could you extract the moin-formatted pages from Experiment/* and put them here?  Also I'd like to take a look at this page: /EnergyCalibrationStandards to see what it says.

Thx

**Comment by bruceravel** [comment]
For example, there is page called Experiment/LytleChamber.  There are a bunch of photos at docs.xrayabsorption.org from this page, but the moin text is not here.

**Comment by newville** [comment]
@bruceravel Yeah, sorry ANL-mandated-and-supplied https certificates are not working out that well. Notices have been sent, some of them polite ;).

The good news is that xafs.org is now redirecting correctly to xafs.xrayabsorption.org.  

I will try to render all the moin pages as HTML and put them to docs.xrayabsorption.org.

**Comment by bruceravel** [comment]
The raw moin text is quite sufficient.  Pandoc works pretty well to translate them.

**Comment by newville** [comment]
millenia appears back up, at least in some jurisdictions.   

Some (I think the most important) HTML pages are now at https://docs.xrayabsorption.org/xafsorg_htmlpages/

FWIW, "raw moin" is not so easy to extract -- each document has revisions and attachments.  Many, many of the pages are not appropriate to move: some spam, many private (especially UOP related).   But I'll try to pull some of those out too.   I don't know if there is a moin2rst, but maybe.  Anyway, it seems like it is actually not so much content and much of it needs a read-through anyway.

**Comment by bruceravel** [comment]
I spent some time looking for a specific moin2rst converter and came up empty.

```
pandoc -f creole -t rst -o foo.rst foo
```
is very close to a proper moin2rst.  Doesn't get everything, but does OK.

I had forgotten how much spam there was on the moin xafs,org.  I am not enough of historian to demand that we preserve the spam :smile:

**Comment by bruceravel** [comment]
I can see millenia again.  as long as that's true, I can fetch moin-markup pages from there.

**Comment by newville** [comment]
The latest moin revisions extracted to  https://docs.xrayabsorption.org/moin_final.tar.gz. 
Shouldn't be too much junk there....
